Dockerfile.build-repo

```diff
@@ -0,0 +1,53 @@
+ARG ALPINE_VERSION=3.7
```

Comment:
This Dockerfile may be top-level.
Old one should be rewritten anyway.

Reply:
ROS package build farm would better having aports repository. Old scripts can be simply removed from this repository.